config/bin_version_strings.cfg

```diff
@@ -295,10 +295,16 @@ iptunnel;;unknown;"iptunnel\ [0-9](\.[0-9]+)+?";"sed -r 's/iptunnel\ ([0-9](\.[0
 ipunz;;commercial;"^ipunz V[0-9]\.[0-9]$";"sed -r 's/ipunz\ V([0-9](\.[0-9]+)+?)$/ipcomm:ipunz:\1/'";
 ipzip;;commercial;"^ipzip V[0-9]\.[0-9]$";"sed -r 's/ipzip\ V([0-9](\.[0-9]+)+?)$/ipcomm:ipzip:\1/'";
 iputils;;unknown;"iputils-s{,3}[0-9]+";"sed -r 's/iputils-s{,3}([0-9]+)$/iputils:\1/'";
-isc-dhclient;;ISC;"Internet\ Systems\ Consortium\ DHCP\ Client\ [0-9](\.[0-9]+)+?([a-z][0-9])?$";"sed -r 's/Internet\ Systems\ Consortium\ DHCP\ Client\ ([0-9](\.[0-9]+)+?([a-z][0-9])?)$/isc:dhcp_client:\1/'";
-isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?)$/isc:dhcp_client:\1/'";
-isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?-([ABPabp]|rc|RC)[0-3]$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?(-([ABPabp]|rc|RC)[0-3])?)$/isc:dhcp_client:\1/'";
-isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?-ESV-R[0-9]$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?(-ESV-R[0-9])?)$/isc:dhcp_client:\1/'";
+isc-dhclient;;ISC;"Internet\ Systems\ Consortium\ DHCP\ Client\ [0-9](\.[0-9]+)+?([a-z][0-9])?$";"sed -r 's/Internet\ Systems\ Consortium\ DHCP\ Client\ ([0-9](\.[0-9]+)+?([a-z][0-9])?)$/dhcp_client:\1/'";
```

Comment:
with https://github.com/e-m-b-a/emba/pull/1166 we should be able to add the isc as the vendor to the sed command. Would you be interested to address this in your PR?

Reply:
ok, adding them back.